exercise/page.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page</title>
+    <link rel="stylesheet" href="page.css">
+</head>
+<body>
+    <main class="card">
+        <div class="picture">
+            <img src="img/test.jpg" alt="">
+        </div>
+        <div>
+            <h4>4 dayes ago</h4>
+            <h1>Post One</h1>
+        </div>
+        <p>
+            Lorem ipsum dolor sit amet
+            consecteur, adipisicing eliy.
+            Quibusdam animi ladoriosam facilis
+            optio unde delectus nam molestiae
+        </p>
```

Comment:
I think it will be bette to make container html tag of these sections
like
```suggestion
        <main>
            <h4>4 dayes ago</h4>
            <h1>Post One</h1>
            <p>
                Lorem ipsum dolor sit amet
                consecteur, adipisicing eliy.
                Quibusdam animi ladoriosam facilis
                optio unde delectus nam molestiae
            </p>
        </main>
```